app/controllers/api/v1/api_keys_controller.rb

```diff
@@ -79,10 +79,10 @@ def otp
   end
 
   def api_key_create_params
-    params.permit(:name, *ApiKey::API_SCOPES)
+    params.permit(:name, *ApiKey::API_SCOPES, :mfa)
```

Comment:
during `gem signin` we [ask user for api key scopes ](https://github.com/rubygems/rubygems/blob/master/lib/rubygems/gemcutter_utilities.rb#L259). We should update it to also ask for mfa enabled option if the user has MFA enabled, which will be passed as mfa param here.
further, please add/update integration test which passes mfa as well.

Reply:
Sounds good! I'll do that next :) I updated the tests in `api_keys_controller_test` for this change.